domains/certificates/RegistryConnector/Worker/Dockerfile

```diff
@@ -18,6 +17,10 @@ FROM build AS publish
 RUN dotnet publish "Worker.csproj" -c Release -o /app/publish /p:UseAppHost=false
 
 FROM base AS final
+
+RUN apk add --no-cache tzdata
```

Comment:
Do we have a timezone issue for logs, or? :)

Reply:
There has been issues with time zone data before when switching to alpine. It is added to be safe.